_includes/CheatSheet.md

```diff
@@ -222,198 +223,184 @@ d.withMonth(Month.NOVEMBER).withDayOfMonth(1); // '2016-11-01'
-
-```javascript
+`TemporalAdjusters` provide compact business logic for date-based temporals such as `LocalDate`, `LocalDateTime` or `ZonedDateTime`.
 
-var d = LocalDate.parse('2016-12-24');
+```JavaScript
+const d = LocalDate.parse("2016-12-24");
 
 // get first/ last day of month
-d.with(TemporalAdjusters.firstDayOfMonth()) // 2016-12-01
-d.with(TemporalAdjusters.lastDayOfMonth())  // 2016-12-31
+d.with(TemporalAdjusters.firstDayOfMonth()); // 2016-12-01
+d.with(TemporalAdjusters.lastDayOfMonth()); //  2016-12-31
 
 // get the next specified weekday
-d.with(TemporalAdjusters.nextOrSame(DayOfWeek.SUNDAY))   // 2016-12-25
-d.with(TemporalAdjusters.nextOrSame(DayOfWeek.SATURDAY)) // 2016-12-24
-d.with(TemporalAdjusters.next(DayOfWeek.SATURDAY))       // 2016-12-31
+d.with(TemporalAdjusters.nextOrSame(DayOfWeek.SUNDAY)); //   2016-12-25
+d.with(TemporalAdjusters.nextOrSame(DayOfWeek.SATURDAY)); // 2016-12-24
+d.with(TemporalAdjusters.next(DayOfWeek.SATURDAY)); //       2016-12-31
 
 // get the first/last weekday of month
-d.with(TemporalAdjusters.lastInMonth(DayOfWeek.SATURDAY)) // 2016-12-31
-d.with(TemporalAdjusters.firstInMonth(DayOfWeek.SATURDAY)) // 2016-12-03
-
+d.with(TemporalAdjusters.lastInMonth(DayOfWeek.SATURDAY)); //  2016-12-31
+d.with(TemporalAdjusters.firstInMonth(DayOfWeek.SATURDAY)); // 2016-12-03
 ```
 
 Find more adjusters in the TemporalAdjusters API documentation.
 
 ## LocalTime
 
-A LocalTime represents a time without time-zone in the ISO-8601 calendar system such as '10:15:30'
+A `LocalTime` represents a time with **no date** and **no time zone** in the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) calendar system, such as '10:15:30'
 
-### Create a LocalTime instance
+### Create a `LocalTime` instance
 
-```javascript
-
-// obtain the current time in the system default timezone, e.g. '10:29:05.743'
+```JavaScript
+// obtain the current time in the system default time zone, e.g. '10:29:05.743'
 LocalTime.now();
 
-// obtain the current time in the utc timezone, e.g. '09:29:05.743'
+// obtain the current time in the UTC time zone, e.g. '09:29:05.743'
 LocalTime.now(ZoneOffset.UTC);
 
-// obtain an instance of LocalTime from an ISO8601 formatted text string
-LocalTime.parse('09:42');      // '09:42'
-LocalTime.parse('09:42:42');   // '09:42:42'
-LocalTime.parse('09:42:42.123'); // '09:42:42.123'
-LocalTime.parse('09:42:42.123456789'); // '09:42:42.123456789'
-
-// obtain an instance of LocalTime from a hour, minute, second and nano value
-LocalTime.of(23, 55) // '23:55'
-LocalTime.of(23, 55, 42) // '23:55:42'
-LocalTime.of(23, 55, 42, 123000000) // '23:55:42.123'
-
+// obtain an instance of LocalTime from an ISO 8601 formatted text string
+LocalTime.parse("09:42"); //              '09:42'
+LocalTime.parse("09:42:42"); //           '09:42:42'
+LocalTime.parse("09:42:42.123"); //       '09:42:42.123'
+LocalTime.parse("09:42:42.123456789"); // '09:42:42.123456789'
```

Comment:
```suggestion
LocalTime.parse('09:42:42.123456789'); // '09:42:42.123456789'
```

Reply:
... i'll stop doing the change requests here ... could you please just use single quotes throughout the file :)